docs/contributor/04-30-ci-cd.md

```diff
@@ -34,34 +34,31 @@ The following CI jobs are part of the development cycle.
-| [`PR Integration / Kubernetes version compatibility test`](https://github.com/kyma-project/api-gateway/blob/main/.github/workflows/pull-integration.yaml#L74) | Executes the main integration test suite for the API Gateway module and checks for any incompatibilities with the latest Kubernetes version.                               |
-| [`PR Integration / Upgrade tests (pull_request)`](https://github.com/kyma-project/api-gateway/blob/main/.github/workflows/pull-integration.yaml#L93)          | Executes the upgrade integration test suite for the API Gateway module and verifies if the existing release can be successfully upgraded to the new version.               |
+| [`PR Integration Release / Wait for image build`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-integration-release.yaml#L22)                  | Waits for the operator image build to finish so that the image-dependent jobs can be executed.                                                                             |
+| [`pull-api-gateway-manager-build`](https://github.com/kyma-project/test-infra/blob/6fa2829ff262d300663335f80f67050d21b1ae5f/prow/jobs/kyma-project/api-gateway/api-gateway-manager-build.yaml#L6) | Builds Kyma API Gateway Operator image and pushes it to the `dev` registry.                                                                                                |
+| [`PR Integration Release / Integration tests`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-integration-release.yaml#L37)                     | Executes the main integration test suite for the API Gateway module on a k3d cluster. The job installs the Istio module using Kyma CLI.                                    |
+| [`PR Integration Release / Kubernetes version compatibility test`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-integration-release.yaml#L54) | Executes the main integration test suite for the API Gateway module and checks for any incompatibilities with the latest Kubernetes version.                               |
+| [`PR Integration Release / Upgrade tests (pull_request)`](https://github.com/kyma-project/api-gateway/blob/3d10f7c7801f514524a267239924fdc889f2be3f/.github/workflows/pull-integration-release.yaml#L71)          | Executes the upgrade integration test suite for the API Gateway module and verifies if the existing release can be successfully upgraded to the new version.               |
 | [`Pull Request / Golang lint check (pull_request)`](https://github.com/kyma-project/api-gateway/blob/main/.github/workflows/pull-request.yaml#L24)            | Performs linting.                                                                                                                                                          |
```

Comment:
We might also want to use permalinks for `Pull Request / Golang lint check (pull_request)`, `Pull Request / Run unit tests (pull_request)` and `pre-api-gateway-presubmit-scanner`.